qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -1,13 +1,29 @@
 import os
 import crypt
 import logging
+from tempfile import mkstemp as tempfile_mkstemp
 from StringIO import StringIO
 from tasks.cephfs.cephfs_test_case import CephFSTestCase
 from tasks.cephfs.fuse_mount import FuseMount
 from teuthology.exceptions import CommandFailedError
 
 log = logging.getLogger(__name__)
 
+def run_cephfs_shell_script(script, mount_x):
```

Comment:
This method can just be a helper in `TestCephFSShell` below like `_cephfs_shell`.

Reply:
It can. But in case of having a class within a class, I would need to invoke this method by writing `TestCephFSShell.run_cephfs_shell_script` which I did not like to it's length. I would even prefer `run_cephfs_shell_script` over `self.run_cephfs_shell_script`.

Lastly, this method isn't functionally dependent on the class TestCephFSShell. Later, perhaps, it can be moved elsewhere very other testsuites can use it.